README.md

```diff
@@ -1,74 +1,55 @@
 # ArkDoc ![release](https://img.shields.io/github/v/release/ArkScript-lang/ArkDoc)
 
-It's a **Lite** documentation generator for arkscript based on Mkdocs, and written in ruby.
-
 ![licence](https://img.shields.io/badge/licence-MPL%202.0-2)
 ![code size](https://img.shields.io/github/languages/code-size/ArkScript-lang/ArkDoc)
-![Ruby](https://github.com/ArkScript-lang/ArkDoc/workflows/Ruby/badge.svg)
+
+A documentation generator for ArkScript.
 
 ## Dependencies
 
-* Python >= 2.7 (Python 3.7 tested and working)
-* Ruby >= 2.5
+* Python >= 3.9
 
 ## Usage
 
 ```bash
-# first, clone repository
-~$ git clone https://github.com/ArkScript-lang/ArkDoc.git
-~$ cd ArkDoc
-# put your arkscript files in this folder (default source directory)
-# you can change it with -s option
-~/ArkDoc$ mkdir source
-# install Mkdocs
-~$ pip install mkdocs
-# running
-~/ArkDoc$ ruby ArkDoc.rb --help
-DESCRIPTION
-      	Lite documentation generator based on Mkdocs
-
-SYNOPSIS
-        ruby ArkDoc.rb -h
-        ruby ArkDoc.rb -v
-        ruby ArkDoc.rb -g <site name> [-s <source path>]
-        ruby ArkDoc.rb -md <site name> [-s <source path>]
-
-OPTIONS
-        -h, --help              Print this help message
-        -v, --version           Print ArkDoc version and exit
-        -g, --generate          Generate the website and exit
-        -md, --markdown         Generate the markdown files and exit
-        -s, --source            Set the path of souree files directory
-
-LICENCE
-        Mozilla Public License 2.0
+git clone https://github.com/ArkScript-lang/ArkDoc.git
+python3 -m venv venv
+source ./venv/bin/activate
+pip3 install -r requirements.txt
+python3 -m arkdoc --help
 ```
 
 ## Syntax
-You can be find the syntax ![here](./Syntax.md).
+
+- `@meta <name>`
```

Comment:
Should we remove @meta since we now use the basename of the file (stripped of its extension)? @ArkScript-lang/lead-developers